lib/app_web/views/comment_view.ex

```diff
@@ -0,0 +1,8 @@
+defmodule AppWeb.CommentView do
+  use AppWeb, :view
+
+  def format_date(date) do
+    NaiveDateTime.truncate(date, :second)
```

Comment:
I think the ```truncate``` function has been added with Elixir 1.6 as there is no documentation for it with Elixir 1.5
https://hexdocs.pm/elixir/1.5.2/NaiveDateTime.html

Also I think the latest version of Elixir on Heroku is 1.5 so I'm not sure we should use ```truncate``` at this stage